lib/trento_web/controllers/v1/tags_controller.ex

```diff
@@ -1,33 +1,85 @@
 defmodule TrentoWeb.V1.TagsController do
   use TrentoWeb, :controller
+  use OpenApiSpex.ControllerSpecs
 
   alias Trento.Tags
 
+  alias TrentoWeb.OpenApi.Schema
+
   action_fallback TrentoWeb.FallbackController
+  plug OpenApiSpex.Plug.CastAndValidate, json_render_error_v2: true
+
+  operation :add_tag,
+    summary: "Add tag",
+    tags: ["Tags"],
+    description: "Add a tag to a resource.",
+    parameters: [
+      id: [
+        in: :path,
+        required: true,
+        type: %OpenApiSpex.Schema{type: :string, format: :uuid}
+      ]
+    ],
+    request_body:
+      {"Tag", "application/json",
+       %OpenApiSpex.Schema{
+         type: :object,
+         properties: %{
+           value: %OpenApiSpex.Schema{type: :string}
+         }
+       }},
+    responses: [
+      accepted: "The tag has been added to the resource",
+      bad_request: Schema.BadRequest.response(),
```

Comment:
Currently this operation never returns a bad request.
However if the plan is to add that case and we want to have that documented in the meantime, ignore the comment.

Reply:
it does, phoenix handles it